lang/cs/Org.Apache.REEF.Client/Common/IDriverHttpEndpoint.cs

```diff
@@ -24,8 +25,21 @@ namespace Org.Apache.REEF.Client.Common
     [Unstable("0.13", "Working in progress for what to return after submit")]
     public interface IDriverHttpEndpoint
     {
+        /// <summary>
+        /// Get Uri result
```

Comment:
The comment is new. We should improve upon it beyond adding spaces to the method name :)